Sources/Commands/Utilities/SymbolGraphExtract.swift

```diff
@@ -107,3 +122,14 @@ public struct SymbolGraphExtract {
         return try process.waitUntilExit()
     }
 }
+
+extension ResolvedModule {
+    func cxxInterop() -> Bool {
```

Comment:
Could you please pick a more descriptive name for this, e.g. `var enablesCxxInterop: Bool { get { ... } }`?

Reply:
To follow Swift API guidelines, I'd prefer this to be named `isCxxInteropEnabled`.